src/app/widgets/inlineinput/inlineinput.component.ts

```diff
@@ -25,7 +25,9 @@ export class InlineInputComponent implements OnInit {
   @Input() type: string = 'string';
   @Input('disabled') readOnly: boolean = false;
   @Input('value') set input(val) {
-    const v = this.convertSpecialChar(val);
+    // convert special charecters only the input value is a string
+    const v = typeof(val) === 'string' ?
```

Comment:
@vikram-raj do you find this change okay?
The type of input for `value` can be anything, but `str.replace` function only works on a string.

Reply:
Yes, It looks good to me.